tests/common/defrag_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"testing"
+	"time"
+
+	"go.etcd.io/etcd/tests/v3/framework"
+
+	"go.etcd.io/etcd/tests/v3/framework/config"
+	"go.etcd.io/etcd/tests/v3/framework/testutils"
+)
+
+func TestDefragOnline(t *testing.T) {
```

Comment:
From what I see TestDefragOnline and TestDefragOffline differ only by options?
Could we have table test `TestDefrag` with 2 subtests `Online` and `Offline`?

Reply:
If not migrating `etcdutl` to common, it will keep online. Offline would be delete.